docs/user/openstack/etcd-ephemeral-disk.md

```diff
@@ -0,0 +1,252 @@
+# Moving etcd to an ephemeral local disk
+
+You can move etcd from a root volume (Cinder) to a dedicated ephemeral local disk to prevent or resolve performance issues.
+
+## Prerequisites
+
+* This migration is currently tested and documented as a day 2 operation.
+* An OpenStack cloud where Nova is configured to use local storage for ephemeral disks. The `libvirt.images_type` option in `nova.conf` must not be `rbd`.
+* An OpenStack cloud with Cinder being functional and enough available storage to accommodate 3 Root Volumes for the OpenShift control plane.
+* OpenShift will be deployed with IPI for now; UPI is not yet documented but technically possible.
+* The control-plane machine’s auxiliary storage device, such as /dev/vdb, must match the vdb. Change this reference in all places in the file.
+
+## Procedure
+
+* Create a Nova flavor for the Control Plane which allows 10 GiB of Ephemeral Disk:
+
+```bash
+openstack flavor create --ephemeral 10 [...]
+```
+
+* We will deploy a cluster with Root Volumes for the Control Plane. Here is an example of `install-config.yaml`:
+
+```yaml
+[...]
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      type: ${CONTROL_PLANE_FLAVOR}
+      rootVolume:
+        size: 25
```

Comment:
25 is not a recommended value, let's use something bigger

Reply:
doc says:

```
* `size` (required integer): Size of the root volume in GB. Must be set to at least 25.
```

And I'm happy to put more but then we need to be consistent in our doc and in the installer validations.

Reply:
25 is the minimum value and is fine for throwaway clusters like we do in CI. For production [we recommend at least 100](https://docs.openshift.com/container-platform/4.14/installing/installing_openstack/installing-openstack-installer-custom.html#installation-osp-control-machines_installing-openstack-installer-custom).

[The validations](https://github.com/openshift/installer/blob/a1ea1c8/pkg/asset/installconfig/openstack/validation/machinepool.go#L19-L20) reflect these values.

I've pushed https://github.com/openshift/installer/pull/7698 to clarify the docs.